### DISCUSSION TITLE
FLUX traning with 16GB VRAM ???

### DISCUSSION THREAD

**Discussion by kalle07** [bmaltais/kohya_ss#2767] [Q&A]
its possible ?

**Comment by FurkanGozukara** [comment]
even 7.5 GB possible : https://youtu.be/nySGu12Y05k

**Reply by kalle07** [reply]
hint:

for me is
turn off "Cache Text Encoder Outputs to Disk"
~15% faster

RTX4060 (16GB)
what you think, when torch 2.5 will be aviable ? one week  one month?

**Reply by FurkanGozukara** [reply]
@kalle07 very nice tips gonna test and update

I heard that torch 2.4.1 solves speed issue and I expect it soon like 1 week

**Reply by kalle07** [reply]
"Split Mode"
dont work for me ... i must unmark it

**Comment by metamountain** [comment]
Could someone post the .json config files for:

8GB
12GB
16GB
24GB

That would be super helpful for the start....  I´´m also having a hard time on my 3090 ;(
